### DISCUSSION TITLE
Welcome to trilium-scripts Discussions!

### DISCUSSION THREAD

**Discussion by justyns** [justyns/trilium-scripts#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sottey** [comment]
My name is Sean and I was a developer in a past life, then a stand up comic, and now I work for the greatest music company in the world, Roland handling Customer Success.

Over the years I have developed in C/C++, C#, Java, PHP, Go and Javascript. Primarily I focus on backend code, as I am unable to ever make anything look pretty.

Currently my focus is integrating things with Trilium. I have gotten Calendar.js working flawlessly as a calendar, storing all event data in a Trilium Note. I have also written a Todo application that isn't quite ready for release as I refuse to release it without recurring tasks. Finally, the amazing Command Palette.... I am racking my brain, coming up with commands. My first being the color picker plugin, which I have always wanted in Trilium.
